configs/genesis-assets/genesis.json

```diff
@@ -0,0 +1,202 @@
+{
```

Comment:
Is this compatible with the genesis loader developed by @greged93 ?

Reply:
yep, is copy pasted from his

Reply:
https://github.com/keep-starknet-strange/madara/tree/main/crates/node/src/genesis_assets